File: VisualStudioPlugins/Sample/SampleLibrary/Composite.cs
namespace Sample
{
    using System;
    using Sample.DataContracts;
    
    
    public partial class Composite
    {
        
        private SummaryDataContract _summary;
        
        private DetailDataContract _detail;
        
        public Composite(SummaryDataContract summary, DetailDataContract detail)
        {
            _summary = summary;
            _detail = detail;
        }
        
        /// <summary>
        /// SummaryDataContract.FirstName
        /// </summary>
        public virtual string FirstName
        {
            get
            {
                return this._summary.FirstName;
            }
            set
            {
                this._summary.FirstName = value;
            }
        }
        
        /// <summary>
        /// SummaryDataContract.LastName
        /// </summary>
        public virtual string LastName
        {
            get
            {
                return this._summary.LastName;
            }
            set
            {
                this._summary.LastName = value;
            }
        }
        
        /// <summary>
        /// SummaryDataContract.Phone
        /// </summary>
        public virtual string Phone
        {
            get
            {
                return this._summary.Phone;
            }
            set
            {
                this._summary.Phone = value;
            }
        }
        
        /// <summary>
        /// SummaryDataContract.ID
        /// </summary>
        public virtual int ID
        {
            get
            {
                return this._summary.ID;
            }
        }
        
        /// <summary>
        /// SummaryDataContract.Created
        /// </summary>
        public virtual System.DateTime Created
        {
            get
            {
                return this._summary.Created;
            }
        }
        
        /// <summary>
        /// SummaryDataContract.Modified
        /// </summary>
        public virtual System.DateTime Modified
        {
            get
            {
                return this._summary.Modified;
            }
        }
        
        /// <summary>
        /// DetailDataContract.Address1
        /// </summary>
        public virtual string Address1
        {
            get
            {
                return this._detail.Address1;
            }
            set
            {
                this._detail.Address1 = value;
            }
        }
        
        /// <summary>
        /// DetailDataContract.Address2
        /// </summary>
        public virtual string Address2
        {
            get
            {
                return this._detail.Address2;
            }
            set
            {
                this._detail.Address2 = value;
            }
        }
        
        /// <summary>
        /// DetailDataContract.City
        /// </summary>
        public virtual string City
        {
            get
            {
                return this._detail.City;
            }
            set
            {
                this._detail.City = value;
            }
        }
        
        /// <summary>
        /// DetailDataContract.State
        /// </summary>
        public virtual string State
        {
            get
            {
                return this._detail.State;
            }
            set
            {
                this._detail.State = value;
            }
        }
        
        /// <summary>
        /// DetailDataContract.Zip
        /// </summary>
        public virtual string Zip
        {
            get
            {
                return this._detail.Zip;
            }
            set
            {
                this._detail.Zip = value;
            }
        }
        
        /// <summary>
        /// DetailDataContract.DetailID
        /// </summary>
        public virtual int DetailID
        {
            get
            {
                return this._detail.ID;
            }
        }
        
        /// <summary>
        /// DetailDataContract.DetailCreated
        /// </summary>
        public virtual System.DateTime DetailCreated
        {
            get
            {
                return this._detail.Created;
            }
        }
        
        /// <summary>
        /// DetailDataContract.DetailModified
        /// </summary>
        public virtual System.DateTime DetailModified
        {
            get
            {
                return this._detail.Modified;
            }
        }
    }
}

File: VSX/Sample/SampleLibraryVB/Composite.vb
Imports System
Imports Sample.DataContracts

Namespace Sample
    
    Partial Public Class Composite
        
        Private _summary As SummaryDataContract
        
        Private _detail As DetailDataContract
        
        Public Sub New(ByVal summary As SummaryDataContract, ByVal detail As DetailDataContract)
            MyBase.New
            _summary = summary
            _detail = detail
        End Sub
        
        '''<summary>
        ''' SummaryDataContract.FirstName
        ''' </summary>
        Public Overridable Property FirstName() As String
            Get
                Return Me._summary.FirstName
            End Get
            Set
                Me._summary.FirstName = value
            End Set
        End Property
        
        '''<summary>
        ''' SummaryDataContract.LastName
        ''' </summary>
        Public Overridable Property LastName() As String
            Get
                Return Me._summary.LastName
            End Get
            Set
                Me._summary.LastName = value
            End Set
        End Property
        
        '''<summary>
        ''' SummaryDataContract.Phone
        ''' </summary>
        Public Overridable Property Phone() As String
            Get
                Return Me._summary.Phone
            End Get
            Set
                Me._summary.Phone = value
            End Set
        End Property
        
        '''<summary>
        ''' SummaryDataContract.ID
        ''' </summary>
        Public Overridable ReadOnly Property ID() As Integer
            Get
                Return Me._summary.ID
            End Get
        End Property
        
        '''<summary>
        ''' SummaryDataContract.Created
        ''' </summary>
        Public Overridable ReadOnly Property Created() As Date
            Get
                Return Me._summary.Created
            End Get
        End Property
        
        '''<summary>
        ''' SummaryDataContract.Modified
        ''' </summary>
        Public Overridable ReadOnly Property Modified() As Date
            Get
                Return Me._summary.Modified
            End Get
        End Property
        
        '''<summary>
        ''' DetailDataContract.Address1
        ''' </summary>
        Public Overridable Property Address1() As String
            Get
                Return Me._detail.Address1
            End Get
            Set
                Me._detail.Address1 = value
            End Set
        End Property
        
        '''<summary>
        ''' DetailDataContract.Address2
        ''' </summary>
        Public Overridable Property Address2() As String
            Get
                Return Me._detail.Address2
            End Get
            Set
                Me._detail.Address2 = value
            End Set
        End Property
        
        '''<summary>
        ''' DetailDataContract.City
        ''' </summary>
        Public Overridable Property City() As String
            Get
                Return Me._detail.City
            End Get
            Set
                Me._detail.City = value
            End Set
        End Property
        
        '''<summary>
        ''' DetailDataContract.State
        ''' </summary>
        Public Overridable Property State() As String
            Get
                Return Me._detail.State
            End Get
            Set
                Me._detail.State = value
            End Set
        End Property
        
        '''<summary>
        ''' DetailDataContract.Zip
        ''' </summary>
        Public Overridable Property Zip() As String
            Get
                Return Me._detail.Zip
            End Get
            Set
                Me._detail.Zip = value
            End Set
        End Property
        
        '''<summary>
        ''' DetailDataContract.DetailID
        ''' </summary>
        Public Overridable ReadOnly Property DetailID() As Integer
            Get
                Return Me._detail.ID
            End Get
        End Property
        
        '''<summary>
        ''' DetailDataContract.DetailCreated
        ''' </summary>
        Public Overridable ReadOnly Property DetailCreated() As Date
            Get
                Return Me._detail.Created
            End Get
        End Property
        
        '''<summary>
        ''' DetailDataContract.DetailModified
        ''' </summary>
        Public Overridable ReadOnly Property DetailModified() As Date
            Get
                Return Me._detail.Modified
            End Get
        End Property
    End Class
End Namespace

